app/assets/javascripts/discourse/app/templates/wizard.gjs

```diff
@@ -0,0 +1,14 @@
+import RouteTemplate from "ember-route-template";
+import hideApplicationFooter from "discourse/helpers/hide-application-footer";
+import DiscourseLogo from "discourse/static/wizard/components/discourse-logo";
+
+export default RouteTemplate(<template>
+  {{hideApplicationFooter}}
+  <div id="wizard-main">
+    <div class="discourse-logo">
+      <DiscourseLogo />
+    </div>
+
```

Comment:
I moved the canvas into the step, which works just as well (since it's `position: fixed` anyway), but avoids having to relying the step back here